docs/3_troubleshooting.md

```diff
@@ -5,14 +5,16 @@ In case you encounter any issues while using ReVanced Manager, please refer to t
 - 💉 Patching fails with an error
 
   Make sure ReVanced Manager is up to date by following [🔄 Updating ReVanced Manager](2_3_updating.md) and select the **Default** button when choosing patches.
-  
+
 - 🚫 App not installed as package conflicts with an existing package
 
-  An existing installation of the app you're trying to patch is conflicting with the patched app. Uninstall the existing app before installing the patched app.
+  An existing installation of the app you're trying to patch is conflicting with the patched app (i.e., the existing version of the app doesn't allow upgrading to the patched version, or a different copy of the app causes a conflict). Uninstall the existing app before installing the patched app.
```

Comment:
Simplifying should reduce introducing new/niche terminology such as "copy of" or "upgrading":

```suggestion
  An existing installation of the app you're trying to patch conflicts with the patched app (i.e., signature mismatch or downgrade). Uninstall the existing app before installing the patched app.
```